demo/Dockerfile

```diff
@@ -29,7 +29,7 @@ USER jenkins
 RUN echo '<settings><mirrors><mirror><id>central</id><url>http://repo.jenkins-ci.org/simple/repo1-cache/</url><mirrorOf>central</mirrorOf></mirror></mirrors><localRepository>/usr/share/jenkins/ref/.m2/repository</localRepository></settings>' > settings.xml
 RUN /usr/local/maven/bin/mvn -s settings.xml -f repo-wc install && /usr/local/maven/bin/mvn -s settings.xml -f repo-wc/sometests -Dmaven.test.failure.ignore clean install
 
-COPY plugins.txt .
+COPY plugins.txt ./
```

Comment:
Not sure how it worked before. Without the slash I get [this](http://stackoverflow.com/questions/26405702/dockerfile-copy-lchown-not-a-directory).